quizcon/templates/main/student_help.html

```diff
@@ -0,0 +1,93 @@
+{% load quiz_tools %}
+<div class="modal fade" id="studentHelp" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="studentHelpLabel" aria-hidden="true">
+    <div class="modal-dialog  modal-dialog-centered modal-lg">
+        <div class="modal-content">
+            <div class="modal-body">
+                <div class="row p-0 m-0">
+                    <div class="col-md-6">
+                        <div id="studentHelpCarouselText" class="studentHelpCarousel carousel slide" data-bs-ride="carousel" data-bs-interval="false" data-bs-wrap="false">
+                            <div class="carousel-inner">
+                                <div class="carousel-item carousel-txt-item text-center align-self-center active">
+                                    <h2>Before You Begin</h2>
+                                    <p>Familiarize yourself with these unique Quizzing With Confidence conventions before taking this quiz.</p>
+                                    <p>Unlike traditional multiple choice questions, the Quizzing With Confidence triangle provides a range of answer possibilities.</p>
+                                </div>
+                                <div class="carousel-item carousel-txt-item text-center align-self-center">
+                                    <h2>Scoring</h2>
+                                    <p>It's not just "correct" and "incorrect" with Quizzing With Confidence. Your score is unique calculation based on both difficulty and confidence that your instructor determines in advance.</p>
+                                    <p>In Quizzing With Confidence, you can even say "I don't know" with Option D.</p>
+                                </div>
+                                <div class="carousel-item carousel-txt-item">
+                                    <h5 class ="text-center">This Quiz</h5>
```

Comment:
I'm wondering if these details are best conveyed on the next screen? There's a little too much detail here.